.eslintrc.js

```diff
@@ -1,102 +1,99 @@
 module.exports = {
```

Comment:
Prettier automatically dropped all the quotes which seems, preferable?

Reply:
sure